recipes/grpc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(WIN32)
+  add_definitions(-D_WIN32_WINNT=0x600)
+endif()
+set(CMAKE_CXX_STANDARD 11)
+
+# Find Protobuf installation
+
+set(protobuf_MODULE_COMPATIBLE TRUE)
+find_package(protobuf CONFIG REQUIRED)
+message(STATUS "Using protobuf ${protobuf_VERSION}")
+
+set(_PROTOBUF_LIBPROTOBUF protobuf::libprotobuf)
+# set(_PROTOBUF_PROTOC $<TARGET_FILE:protobuf::protoc>)
+find_program(_PROTOBUF_PROTOC protoc ${CONAN_BIN_DIRS_PROTOBUF} NO_DEFAULT_PATH)
```

Comment:
```suggestion
```

Reply:
Hi! `${Protobuf_PROTOC_EXECUTABLE}` is provided by the `find_package(protobuf)` which is always the _host_ package. In a cross-compiler scenario that _host_ package cannot provide a binary that runs locally which is what we need here.

`find_program(_PROTOBUF_PROTOC protoc ...)` has a better chance here.... if we use whatever is in the PATH instead of `${CONAN_BIN_DIRS_PROTOBUF}` (which is the PATH to _host_ package). From the outside, Conan is able to add to the PATH the directory where `protoc` (from a build-requires) is located.

---

Nevertheless, there is a lot of work in progress regarding this issue, so there is no standard solution. If it works, I would prefer not to lock it to the _native_ compilation... but if you agree it is better the other way I will change it.

Reply:
@jgsogo protobuf recipe properly handles this case now: https://github.com/conan-io/conan-center-index/blob/1e219c43aacd6c8b30b697d1f95e84dd9cce29b7/recipes/protobuf/all/conanfile.py#L120-L142

Reply:
We need to add examples/READMEs/info to the detail view of packages in ConanCenter. This can be useful (https://conan.io/center/protobuf), but there is more information that could be much more useful 🚀